apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1.sln

```diff
@@ -60,4 +60,7 @@ Global
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		EnterpriseLibraryConfigurationToolBinariesPathV6 = packages\EnterpriseLibrary.TransientFaultHandling.6.0.1304.0\lib\portable-net45+win+wp8
```

Comment:
Using Topaz to test transaction retries.  This will go away when I do the final integrate to master.
We should talk about overall structure at some point and whether it makes sense to move data to its own folder since I will soon need to reference the admin vkit libraries.